crate-features/Cargo.toml

```diff
@@ -21,7 +21,7 @@ rand_chacha = { version = "0.1.1" }
 regex-syntax = { version = "0.6.12" }
 reqwest = { version = "0.10.1", default-features = false, features = ["blocking", "rustls-tls", "json"] }
 serde = { version = "1.0.100", features = ["rc"] }
-ed25519-dalek = { version = "=1.0.0-pre.1", features = ["serde"] }
+ed25519-dalek = { version = "=1.0.0-pre.3", features = ["serde"] }
```

Comment:
Do we need to bump `curve25519-dalek` (above, can't comment there) to `"2"` as per the `ed25519-dalek` `Cargo.toml` ?